src/evict/evict.h

```diff
@@ -179,7 +179,7 @@ static WT_INLINE bool __wt_evict_clean_pressure(WT_SESSION_IMPL *session)
 static WT_INLINE bool __wt_evict_dirty_needed(WT_SESSION_IMPL *session, double *pct_fullp)
   WT_GCC_FUNC_DECL_ATTRIBUTE((warn_unused_result));
 static WT_INLINE bool __wt_evict_needed(WT_SESSION_IMPL *session, bool busy, bool readonly,
-  double *pct_fullp) WT_GCC_FUNC_DECL_ATTRIBUTE((warn_unused_result));
+  bool ignore_updates_dirty, double *pct_fullp) WT_GCC_FUNC_DECL_ATTRIBUTE((warn_unused_result));
```

Comment:
Instead of hardcoding this temporarily, why not provide through a debug config, which can be turned on/off.

Reply:
We need to use this change to run some testing on standby. This behaviour should be automatic when the role of the node change. Adding a config complicates this as we need a server change to facilitate that.